### PR TITLE
Speaker Profiles 

### PR DESCRIPTION
**Pull request by MrSunshyne**
Everyone gets speaker profiles !
This applies retroactively to all meetups we've ever done.
All pictures of the speakers are taken from their github account profile picture

image

image

image

### CODE REVIEW

File: src/components/SpeakerSingle.astro [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+<div>
+  <!-- <pre>
+        <code>{JSON.stringify(props.speaker, null, 2)}</code>
+    </pre> -->
+  <div data-title={props.speaker.person.name}>
+    <div class="relative">
+      <div class="pb-10 md:pb-24">
+        <div class="contain">
+          <!-- <div class="">
```
Comment (Saamiyah) [inline src/components/SpeakerSingle.astro:22]: commented code can be removed

File: src/utils/api.ts [hunk 2]
```diff
@@ -38,3 +34,27 @@ export async function getEvent(id: string | number) {
...
+    const speakers = await loadSpeakers();
+    const speaker = speakers.data.find(ev => ev.id == id);
+    const events  = await loadEvents()
+    
+    if (speaker === null) {
+        console.error("cannot find speaker id: ", id);
+    }
+
+    // Get sessions of this speaker from the events
+    const allSessions = events.data.map(event => event.sessions).flat()//.filter(session => session.speakers.find(speaker => speaker.id == id))
```
Comment (Saamiyah) [inline src/utils/api.ts:48]: remove unused filter on line 48

File: src/components/SpeakerSingle.astro (hunk 1 above)
Comment (MrSunshyne) [reply src/components/SpeakerSingle.astro:22]: commented code removed

File: src/utils/api.ts (hunk 2 above)
Comment (MrSunshyne) [reply src/utils/api.ts:48]: done

**Review by Saamiyah — APPROVED** [review]
Great job on the speaker profiles! Looking amazing ! 🙌

**Pull request merged**